setup.py

```diff
@@ -44,6 +50,7 @@ def _load_description() -> str:
         "uvicorn",
         "starlette_exporter",
         "py-grpc-prometheus",
+        "uvloop;" + env_marker_cpython,
```

Comment:
Interesting! I didn't know that `uvloop` wasn't supported in Win. Thanks for adding the guard!